### ISSUE TITLE
[Adaptive Card iOS - ProductVideo.json]: Voice over is conveying irrelevant information while navigating on the 'Adaptive Cards Product Video' heading.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#170]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Scenarios' control and activate it.
5. Navigate to 'VideoPlayer.json' under 'Scenarios' list item and activate it.
6. Card will be appear.
7. Perform right swipe and navigate to the video player and double tap to activate it.
8. Now, navigate to the 'Adaptive Cards Product Video' and observe the issue.

**Actual Result:**
While navigating on the Adaptive cards product video heading, screen reader is conveying the irrelevant information as double tap to activate it as on double tapping on it no action is getting performed.

**Expected Result:**
Voice over should not convey the irrelevant information for the adaptive card product video heading. It should not announce as double tap to activate it.

**User Impact:**
It will impact the screen reader user if the voice over will convey irrelevant information as double tap to edit for the non editable field as they will try to perform action on it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/3c6bcbc2-322c-49c7-9b73-971507796027

### Card JSON

```JSON
NA
```
